src/routes/+page.svelte

```diff
@@ -1,79 +1,82 @@
 <script lang="ts">
-  import "./page.scss";
-  import ContentfulRichText from "$lib/components/ContentfulRichText";
-  // import VideoCard from "$lib/components/VideoCard";
+  import "./(infoPages)/[topTierPage]/page.scss";
 
-  import {
-    src as underConstructionGif,
-    width as underConstructionWidth,
-    height as underConstructionHeight,
-  } from "$lib/assets/under-construction.gif?as=metadata&imagetoolsMetadata";
+  import { url as arrowIcon } from "$icons/arrow_forward";
 
-  // vite-imagetools doesn't support converting to an animated avif on the fly, so i did it ahead of
-  // time and added it to the repo. it's only 12kb.
-  import underConstructionAvif from "$lib/assets/under-construction.avif";
+  import { getSources } from "$lib/imageServices/contentful";
 
-  import landingImage from "$lib/assets/commissioner-strain-with-farmer.jpg?quality=85&imagetools";
-  import landingImageMobile from "$lib/assets/commissioner-strain-with-farmer.jpg?quality=85&w=412&imagetools";
-  import landingImageWebP from "$lib/assets/commissioner-strain-with-farmer.jpg?format=webp&quality=85&imagetools";
-  import landingImageWebpMobile from "$lib/assets/commissioner-strain-with-farmer.jpg?format=webp&quality=85&w=412&imagetools";
-  import landingImageAvif from "$lib/assets/commissioner-strain-with-farmer.jpg?format=avif&quality=85&imagetools";
-  import landingImageAvifMobile from "$lib/assets/commissioner-strain-with-farmer.jpg?format=avif&quality=85&w=412&imagetools";
-  import landingImageBlurhash, {
-    width as landingImageWidth,
-    height as landingImageHeight,
-    mean as landingImageMean,
-  } from "$lib/assets/commissioner-strain-with-farmer.jpg?blurhash";
-
-  import Image from "$lib/components/Image";
+  import Button, { type Variant } from "$lib/components/Button";
+  import Card from "$lib/components/Card";
+  import Icon from "$lib/components/Icon";
+  import VideoCard from "$lib/components/VideoCard";
+  import Image from "$lib/components/Image/Image.svelte";
 
   export let data;
+  $: ({
+    homePage: { heroVideo, featuredServices },
+  } = data);
+
+  const getCardSettings = (
+    index: number
+  ): { card: "full" | "half" | "third"; button: Variant; imageLoading: "lazy" | "eager" } => {
```

Comment:
For the first draft we're not doing anything fancy with the card layout, but I figured it would be useful later.